crates/autopilot/src/solvable_orders.rs

```diff
@@ -107,6 +110,7 @@ impl SolvableOrdersCache {
         limit_order_price_factor: BigDecimal,
         protocol_fees: domain::ProtocolFees,
         cow_amm_registry: cow_amm::Registry,
+        run_loop_mode: RunLoopMode,
```

Comment:
I wonder if we can keep the run_loop_mode local to the run_loop component? ie. make the update method public and call it inside `next_auction` if we are in `SyncToBlockchain` mode?

Reply:
Yeah, good point.
Initially I didn't try too hard to keep the changes maximally local to the runloop since I expect that we will get rid of these ifs very soon anyway. But it turned our to not be a big change after all. 👌